docs/autogen.py

```diff
@@ -53,19 +53,15 @@ def get_function_signature(function, method=True):
         args = args[: -len(defaults)]
     else:
         kwargs = []
-    st = "%s.%s(" % (clean_module_name(function.__module__), function.__name__)
-
-    for a in args:
-        st += str(a) + ", "
-    for a, v in kwargs:
-        if isinstance(v, str):
-            v = "'" + v + "'"
-        st += str(a) + "=" + str(v) + ", "
-    if kwargs or args:
-        signature = st[:-2] + ")"
-    else:
-        signature = st + ")"
-    return signature
+    signature = ["{}.{}(".format(clean_module_name(function.__module__), function.__name__)]
+
+    for arg in args:
+        signature.append(str(arg))
+    for key, value in kwargs:
+        if isinstance(value, str):
+            value = "'{}'".format(value)
+        signature.append("{}={}".format(key, value))
+    return ', '.join(signature) + ")"
```

Comment:
given that we finally released the last python2 version, please use fstrings in these changes

e.g. `value = f"'{value}'"